docs/docs.yml

```
@@ -188,34 +188,38 @@ navigation:
         contents:
           - section: Send
             contents:
-              - page: EVM
+              - page: Send A Transaction
                 path: wallets/pages/transactions/send/evm-user-ops.mdx
-              - page: Solana
+              - page: Batch Send Transactions
+                path: wallets/pages/transactions/send/batch-user-ops.mdx
+              - page: Retry Transactions
+                path: wallets/pages/transactions/retry-user-operations.mdx
+              - page: Solana Support
                 path: wallets/pages/transactions/send/solana-user-ops.mdx
-          - page: Batch
-            path: wallets/pages/transactions/send/batch-user-ops.mdx
-          - section: Sponsor gas
+              - section: Using 7702
+                contents:
+                  - page: Overview
+                    path: wallets/pages/react/using-7702.mdx
+                  - page: Upgrade existing embedded EOAs
+                    path: wallets/pages/smart-contracts/modular-account-v2/using-7702.mdx
+                  - section: 7702 Quickstart
+                    contents:
+                      - page: Intro
+                        path: wallets/pages/smart-wallets/7702-quickstart/index.mdx
+                      - page: Using SDK
+                        path: wallets/pages/smart-wallets/7702-quickstart/sdk.mdx
+                      - page: Using API
+                        path: wallets/pages/smart-wallets/7702-quickstart/api.mdx
+          - section: Sponsor Gas
             contents:
-              - page: EVM
+              - page: Sponsor UserOps
```

No user op language hehe
```suggestion
              - page: Sponsor gas
```